sympy/core/assumptions_generated.py

```diff
@@ -0,0 +1,1443 @@
+                                                    11,
+                                                    14,
+                                                    15,
+                                                    16,
+                                                    17,
+                                                    21,
+                                                    25,
+                                                    28,
+                                                    35,
+                                                    36],
+                   ('extended_nonzero', False): [7, 9, 13, 14, 15, 17, 21, 22],
+                   ('extended_nonzero', True): [4,
+                                                5,
+                                                6,
+                                                8,
+                                                10,
+                                                11,
+                                                12,
+                                                13,
+                                                14,
+                                                15,
+                                                16,
+                                                18,
+                                                27,
+                                                35],
+                   ('extended_positive', False): [7, 8, 9, 13, 15, 16, 17, 21],
+                   ('extended_positive', True): [4,
+                                                 5,
+                                                 6,
+                                                 15,
+                                                 18,
+                                                 22,
+                                                 23,
+                                                 26,
+                                                 27,
+                                                 35],
+                   ('extended_real', False): [],
+                   ('extended_real', True): [4, 5, 6, 8, 9, 10, 13, 14, 35, 36],
+                   ('finite', False): [7, 8, 10, 35, 36],
+                   ('finite', True): [3, 4, 6, 23, 24, 25, 26, 27],
+                   ('hermitian', False): [3, 5, 7, 8, 10, 35, 36],
+                   ('imaginary', True): [0],
+                   ('infinite', False): [3, 4, 6, 23, 24, 25, 26, 27],
+                   ('infinite', True): [7, 8, 10, 35, 36],
+                   ('integer', False): [7, 8, 10, 15, 16, 18, 35, 36],
+                   ('integer', True): [1,
+                                       2,
+                                       8,
+                                       9,
+                                       10,
+                                       13,
+                                       14,
+                                       16,
+                                       17,
```

Comment:
I don't understand why this is printed so strangely.

Reply:
A human will need to look at the diff any time the implications are changed.

Reply:
True, but that is not often. The formatting looks reasonable to me now. If you prefer another formatting, let me know which format it should be.